src/api/app/components/workflow_run_detail_component.html.haml

```diff
@@ -1,33 +1,27 @@
 %ul.nav.nav-tabs#workflow-run-tabs{ role: 'tablist' }
   %li.nav-item{ role: 'presentation' }
-    %a.nav-link.active{ id: "request-tab#{id}", data: { toggle: 'tab' }, href: "#request-tab-content#{id}",
-                        role: 'tab', aria: { controls: "request-tab-content#{id}", selected: 'true' } }
-      Request
-  %li.nav-item{ role: 'presentation' }
-    %a.nav-link{ id: "response-tab#{id}", data: { toggle: 'tab' }, href: "#response-tab-content#{id}",
-                  role: 'tab', aria: { controls: "response-tab-content#{id}", selected: 'false' } }
-      Response
+    %a.nav-link.active{ id: "incoming-webhook-tab#{id}", data: { toggle: 'tab' }, href: "#incoming-webhook-tab-content#{id}",
+                        role: 'tab', aria: { controls: "incoming-webhook-tab-content#{id}", selected: 'true' } }
+      Incoming Webhook
```

Comment:
Do we know the type of SCM at this point?

Reply:
Yes, we do

Reply:
Then I would suggest `Webhook from $SCM`